frontend/src/components/Gallery/Gallery.jsx

```diff
@@ -0,0 +1,26 @@
+import { useGlobalContext } from "../Context/GlobalContextProvider";
+import "./Gallery.scss";
+import ArtworkCard from "./ArtworkCard";
+
+function Gallery() {
+  const { artwork } = useGlobalContext();
+  return (
+    <main className="galleryContainer">
```

Comment:
Refactoring is possible here, to cut into 3 components

Reply:
new component mapped